apiary.apib

```diff
@@ -1706,22 +1706,802 @@ root will not be listable, as it is virtual. Access known paths directly.
 ## bw
 
 # Group swarm
+Swarm inspection tool.
 
-## addrs
+This command can't be called directly.
```

Comment:
Is this needed? As there is no url attached to it, it's pretty clear that this is not a command that can be called I would say